components/service-binding-usage-controller/cmd/controller/main.go

```diff
@@ -180,20 +180,21 @@ func runHTTPServer(stop <-chan struct{}, addr string, sbuClient *bindingUsageCli
 func informerAvailability(sbuClient *bindingUsageClientset.Clientset, log logrus.FieldLogger, namespace string) error {
 	deleteSample := func() {
 		err := sbuClient.ServicecatalogV1alpha1().ServiceBindingUsages(namespace).Delete(
+			context.TODO(),
```

Comment:
there is a context created already in `runHTTPServer` method, you can send it as a parameter and reuse, here and in other places